params.go

```diff
@@ -434,6 +412,55 @@ func (p *Params) SetDefaults(appLabel, buildVersion, releaseName, releaseAction
 	}
 }
 
+func initializeSidecarDefaults(sidecar *SidecarParams, params *Params) {
+	if sidecar.Image == "" {
+		switch sidecar.Type {
+		case "openresty":
+			sidecar.Image = "estafette/openresty-sidecar:1.13.6.2-alpine"
+		case "cloudsqlproxy":
+			sidecar.Image = "gcr.io/cloudsql-docker/gce-proxy:1.12"
+		}
+	}
+
+	if sidecar.HealthCheckPath == "" {
```

Comment:
I would move all defaults inside the switch so we can use different defaults per sidecar, also for generic properties like memory and cpu.